src/SnmpLibrary/library.py

```diff
@@ -97,7 +99,8 @@ def open_snmp_v3_connection(self, host, user, password='',
                                 encryption_password=None,
                                 authentication_protocol=None,
                                 encryption_protocol=None, port=161,
-                                timeout=1.0, retries=5, alias=None):
+                                timeout=1.0, retries=5, alias=None,
+                                context_name=null):
```

Comment:
could you please add the description into the documentation section below

Reply:
Done in 0f18d150a6ac21aaf7dcf4dfddbf732bcacc3ace